single-work.php

```diff
@@ -0,0 +1,74 @@
+<?php
+get_header();
+?>
+
+<main id="main-content">
+  <section id="posts">
+
+<?php
+if (have_posts()) {
+  while (have_posts()) {
+    the_post();
+
+    $work_cats = wp_get_post_terms($post->ID, 'work_cat');
+    $work_artists = get_post_artists($post->ID);
+    $work_details = get_post_meta($post->ID, '_igv_work_details', true);
+?>
+
+      <article <?php post_class(); ?> id="post-<?php the_ID(); ?>">
+        <div class="container">
+          <div class="grid-row margin-bottom-basic">
+<?php
+      if (!empty($work_cats)) {
+?>
+            <div class="grid-item item-s-12 font-sans margin-bottom-basic"><?php echo $work_cats[0]->name; ?></div>
+<?php
+      }
+?>
+            <div class="grid-item item-s-10 item-l-6">
+              <h1 class="font-serif font-italic"><?php echo get_the_title($post->ID); ?></h1>
+
+<?php
+      if (!empty($work_artists)) {
+?>
+              <div class="font-serif"><?php echo $work_artists; ?></div>
+<?php
+      }
+?>
+            </div>
+            <div class="grid-item item-s-2 item-l-6 text-align-right">
+              Inquire
+            </div>
+          </div>
+
+          <div class="grid-row margin-bottom-basic">
+            <div class="grid-item item-s-12 item-l-6 font-sans">
+              <?php echo !empty($work_details) ? apply_filters('the_content', $work_details) : '&nbsp;'; ?>
```

Comment:
What is the reason a nbsp is rendered if this is empty? The grid should still work without any content in the div I think?

Reply:
grid-items have flex-grow off by default. so if theyre empty they shrink down to nothing. the alternative here is adding `flex-grow` class to the grid item

Reply:
I think probably the grid shouldn't work like that but id favor an extra class over the nbsp